### PR TITLE
Fix translation mistake in zh/part6

### PR DESCRIPTION
**Pull request by yiyizym**
In part6 of English version, on *6.5 Header compression* topic, we have:
> While the number of objects per web page has increased (as mentioned earlier), the use of cookies and the size of the requests have also kept growing over time. 

This has been translated into Chinese as:
> 当每个页面资源的个数上升的时候，cookies和请求的大小都会增加。

In my opinion, It'd better be translated into Chinese as:
> 每个页面请求的对象数量在增多（如前所述），同时 cookies 的使用和请求的大小也在日渐增长。

### CODE REVIEW

**Comment by bagder** [comment]
@ye11ow, do you have chance to give this a 👍 or 👎 ? I prefer to get at least one more voice on a chance like this where I don't know the language.

**Comment by ye11ow** [comment]
@bagder Yes, of course. I'll review this in the weekend :)

File: zh/part6.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ HTTP是一种无状态的协议。简而言之，这意味着每个请求必须
 
 这也保证了HTTP可重复性。当一个客户端从同一服务器请求了大量资源（例如页面的图片）的时候，所有这些请求看起来几乎都是一致的，而这些大量一致的东西则正好值得被压缩。
 
-当每个页面资源的个数上升的时候，cookies和请求的大小都会增加。cookies需要被包含在所有请求中，且他们在多个请求中经常是一模一样的。
+每个页面请求的对象数量在增多（如前所述），同时 cookies 的使用和请求的大小也在日渐增长。cookies需要被包含在所有请求中，且他们在多个请求中经常是一模一样的。
```
Comment (ye11ow) [inline zh/part6.md]: I prefer "资源" instead of "对象" here, even the English version is "objects"

Comment (yiyizym) [inline zh/part6.md]: I agree with you.

Comment (ye11ow) [inline zh/part6.md]: @yiyizym would you please update your PR so we can merge this? Thanks.

**Comment by yiyizym** [comment]
@ye11ow done.

**Comment by bagder** [comment]
Thanks!

**Pull request merged**